File: src/api/Models/GameStatus.cs
namespace SupercellUilityApi.Models
{
    public class GameStatus
    {
        public string GameName { get; set; }
        public int Status { get; set; }
        public string LatestFingerprintSha { get; set; }
        public string LatestFingerprintVersion { get; set; }
        public long LastUpdated { get; set; }
    }
}

File: src/client/lib/api/models/GameStatus.dart
class GameStatus {
  String gameName;
  int status;
  String latestFingerprintSha;
  String latestFingerprintVersion;
  int lastUpdated;

  GameStatus();

  GameStatus.fromJson(Map<String, dynamic> json)
      : gameName = json["gameName"],
        status = json["status"],
        latestFingerprintSha = json["latestFingerprintSha"],
        latestFingerprintVersion = json["latestFingerprintVersion"],
        lastUpdated = json["lastUpdated"];
}
